cls/SourceControl/Git/Extension.cls

```diff
@@ -43,7 +44,7 @@ XData Menu
 Method UserAction(Type As %Integer, Name As %String, InternalName As %String, SelectedText As %String, ByRef Action As %String, ByRef Target As %String, ByRef Msg As %String, ByRef Reload As %Boolean) As %Status
 {
     #dim ec as %Status = $$$OK
-    #dim menu as %Status = $piece(Name, ",", 1)
+    #dim menu as %Status = $piece(Name, ",", 1)    
     if menu '= "%SourceMenu", menu'="%SourceContext" {
```

Comment:
Prior to this, need to re-route Type 1 / Name 5 to be %SourceMenu,Status

Reply:
When I logged the value of `Name` it was never a number.

Reply:
See docs in %Studio.Extension.Base class reference:

/// <li>1 : Other Studio action</li></ul>
/// <li>5 : User has connected to a new namespace</li>